device/common/include/traccc/seeding/device/make_triplet_counter_buffer.hpp

```diff
@@ -0,0 +1,35 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Library include(s).
+#include "traccc/edm/device/triplet_counter.hpp"
+
+// VecMem include(s).
+#include <vecmem/memory/memory_resource.hpp>
+#include <vecmem/utils/copy.hpp>
```

Comment:
At this point I'm **super** pedantic, but let's add an include for both `<vector>` and `<cstddef>` to this header. Just to make sure that the `std::vector<std::size_t>` type would be understood. Even if upstream headers change their own includes.